dashboards/usage-report.jsonnet

```diff
@@ -57,7 +53,45 @@ local memoryUsageUserPods = barGaugePanel.new(
         }
       ) by (namespace, pod)
     |||,
-    legendFormat='{{namespace}}-{{label_hub_jupyter_org_username}}',
+    legendFormat='{{label_hub_jupyter_org_username}} ({{namespace}})',
+  ),
+]);
+
+# Dask-related
+local memoryUsageDaskWorkerPods = barGaugePanel.new(
+  'Dask-gateway worker pod memory usage',
+  datasource='$PROMETHEUS_DS',
+  unit='bytes',
+  thresholds=[
+    {
+      value: 0,
+      color: 'green',
+    },
+    {
+      value: 600,
```

Comment:
Let's leave these thresholds empty for now? 600 is a bit of an arbitrary value, right?

Reply:
yes, mostly arbitrarily. I added the thresholds to make the dashboard easier to read. No thresholds will produce a dashboard like the one in the initial comment (all grey, hard to distinguish what's what) and this is a dashboard with thresholds:
<img width="1374" alt="Screenshot 2022-06-15 at 09 41 12" src="https://user-images.githubusercontent.com/7579677/173760565-fa670dd6-9cab-4ba9-929f-8ba6e8f5d631.png">


We could keep only the base threshold? and have the bars be one color?

Reply:
uuuh, I like it on pink also

<img width="1346" alt="Screenshot 2022-06-15 at 10 39 32" src="https://user-images.githubusercontent.com/7579677/173770726-1d2841eb-9c1c-44c8-a3e4-44dbb6e1be90.png">

(*ups, I think I'm down a rabbit hole and now I should probably stop playing with colors* 😅 )